app/resources/locale/en/en.po

```diff
@@ -796,129 +793,127 @@ msgstr "New"
 
 #: app/views/design.html:87
 msgid "New Color:"
-msgstr ""
+msgstr "New Color:"
 
 #: app/views/design.html:80
-#, fuzzy
-msgid "New Name:"
-msgstr "Name"
+msgid "New Name"
+msgstr "New Name"
+
+#: app/views/design.html:71
+msgid "Next"
+msgstr "Next"
 
-#: app/scripts/controllers/menu.js:1436
+#: app/scripts/controllers/menu.js:1453
 msgid "No collections stored"
 msgstr "No collections stored"
 
-#: app/scripts/app.js:58 app/scripts/controllers/menu.js:585
-#: app/scripts/services/project.js:146
+#: app/scripts/app.js:57 app/scripts/controllers/menu.js:601
+#: app/scripts/services/project.js:157
 msgid "OK"
 msgstr "OK"
 
 #: app/scripts/services/forms.js:427
-#, fuzzy
 msgid "Olive Drab"
-msgstr "OliveDrab"
+msgstr "Olive Drab"
 
-#: app/views/design.html:208
+#: app/views/design.html:206
 msgid "OliveDrab"
 msgstr "OliveDrab"
 
 #: app/views/menu.html:100
 msgid "Open"
 msgstr "Open"
 
-#: app/scripts/services/utils.js:697
+#: app/scripts/services/utils.js:692
 msgid "Open SVG"
 msgstr "Open SVG"
 
 #: app/scripts/services/forms.js:403
-#, fuzzy
 msgid "Orange Red"
-msgstr "OrangeRed"
+msgstr "Orange Red"
 
-#: app/views/design.html:144
+#: app/views/design.html:143
 msgid "OrangeRed"
 msgstr "OrangeRed"
```

Comment:
Confused about the duplication of OrangeRed and OliveDrab above - I leave it to you

Reply:
I didn't edit any of this (only indentation), but I'll check them now.